<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QDialog {
                background-color: #f7f9f9;
            }
            QLabel {
                font-family: 'Segoe UI', sans-serif;
                font-size: 15px;
                color: #000000;
                font-weight: bold;
            }
            QLineEdit, QComboBox {
                padding: 10px;
                border: 2px solid #03254c;
                border-radius: 8px;
                font-size: 15px;
                background-color: white;
                font-family: 'Segoe UI', sans-serif;
            }
            QLineEdit:focus, QComboBox:focus {
                border: 2px solid #03254c;
                outline: none;
            }
            QPushButton {
                background-color: #1167b1;
                color: white;
                padding: 10px 15px;
                border-radius: 8px;
                font-size: 14px;
                font-weight: bold;
                border: none;
                font-family: 'Segoe UI', sans-serif;
            }
            QPushButton:hover {
                background-color: #2a9df4;
            }
            QPushButton:pressed {
                background-color: #d0efff;
            }
            QDialogButtonBox QPushButton {
                min-width: 80px;
            }
            QDialogButtonBox QPushButton#OKButton {
                background-color: #1167b1;
            }
            QDialogButtonBox QPushButton#OKButton:hover {
                background-color: #2a9df4;
            }
            QDialogButtonBox QPushButton#CancelButton {
                background-color: #1167b1;
            }
            QDialogButtonBox QPushButton#CancelButton:hover {
                background-color: #2a9df4;
            }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="judul_label">
       <property name="text">
        <string>Judul :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_judul"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="penulis_label">
       <property name="text">
        <string> Penulis :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_penulis"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="kategori_label">
       <property name="text">
        <string>kategori :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="edit_kategori">
       <item>
        <property name="text">
         <string>Fiksi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non-Fiksi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pelajaran</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teknologi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sejarah</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biografi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seni</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lainnya</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lokasi_rak_label">
       <property name="text">
        <string>Lokasi Rak: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_lokasi_rak"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
